src/createStore.js

```diff
@@ -92,10 +93,20 @@ export default function createStore(reducer, initialState) {
       isPlainObject(action),
       'Actions must be plain objects. Use custom middleware for async actions.'
     );
+    invariant(
+      !isDispatching,
+      'Cannot dispatch in the middle of a dispatch.'
```

Comment:
Maybe change the wording `Cannot dispatch in the middle of a reducer call.`? To clarify this isn't exactly the same thing as what Flux has.